rtcclient/client.py

```diff
@@ -107,23 +114,25 @@ def _get_headers(self):
             raise exception.RTCException("Authentication Failed: "
                                          "Invalid username or password")
 
+        _cookies = None
```

Comment:
L97 `_headers = {"Content-Type": self.CONTENT_XML}` is useless for now.

Reply:
Thank you for your review. I've modified the source code.